src/protocol.cpp

```diff
@@ -47,47 +47,6 @@ const char* WTXIDRELAY = "wtxidrelay";
 const char* SENDTXRCNCL = "sendtxrcncl";
 } // namespace NetMsgType
 
-/** All known message types. Keep this in the same order as the list of
```

Comment:
There is a comment in src/protocol.h#56 which could be updated to reflect the new constant too:

```cpp
/**
 * Bitcoin protocol message types. When adding new message types, don't forget
 * to update allNetMessageTypes in protocol.cpp.
 */
```

Reply:
Updated, thanks!